parser/src/index.ts

```diff
@@ -1,29 +1,122 @@
 import "dotenv/config"
+import { createSpinner } from "nanospinner"
+import { resolve } from "path"
+import { performance } from "perf_hooks"
+import parser from "yargs-parser"
```

Comment:
Could we maybe call this arg-parser or something. There are a alot of things called "parser" :D